designer/server/src/plugins/routes/api.js

```diff
@@ -0,0 +1,92 @@
+import { Schema } from '@defra/forms-model'
```

Comment:
I've exported the old name for compatibility, but once https://github.com/DEFRA/forms-designer/pull/89 merges we should (hopefully) get typed return values from `formDefinitionSchema.validate()`

```suggestion
import { formDefinitionSchema } from '@defra/forms-model'
```